tests/ml_commons/test_ml_commons_client.py

```diff
@@ -165,3 +165,21 @@ def test_integration_model_train_upload_full_cycle():
                 except:  # noqa: E722
                     raised = True
                 assert raised == False, "Raised Exception in deleting model"
+
+                try:
+                    search_model_obj = ml_client.search_model()
```

Comment:
We don't have to add test here. We can add separate test method for search api testing. This method was mostly covering the full cycle of the model integration.

In the search api we should check how many models we got from the search result. For example if we think there should be one "TEXT_EMBEDDING" model, we should count the output of the search result and assert if the number matches or not.

Reply:
Ok, thank you! I will fix it.